### DISCUSSION TITLE
In Windows 10, how can you capture a rectangular part of the screen selected by the mouse?

### DISCUSSION THREAD

**Discussion by nissansz** [python-pillow/Pillow#7538] [Q&A] [answered]
I want to use pillow in win10 to capture a rectangle part of screen selected by mouse, any sample code?

**Comment by radarhere** [comment, marked as answer]
You won't be able to use Pillow to get the mouse co-ordinates.

You can use ImageGrab.grab to capture part of the screen once you have the coordinates. 

```python
from PIL import ImageGrab
ImageGrab.grab((0, 0, 1, 1))
```

See https://pillow.readthedocs.io/en/stable/reference/ImageGrab.html#PIL.ImageGrab.grab for more info
